stack.yaml

```diff
@@ -229,6 +229,11 @@ extra-deps:
   subdirs:
     - http2-client-grpc
 
+# Fix in branch header-encoding-0-size-table-backport for issue #27
+# Note: the patch for #27 is based on version 2.0.6 of http2
```

Comment:
Can you please mention the branch name here?

Reply:
You mean instead of the SHA?

Reply:
No, you have to write sha in there. But we usually also write the branch name in the comment so if our fork needs any maintenance it is easy to figure out where to do that maintenance.

Reply:
For eample: https://github.com/wireapp/wire-server/blob/3ad1409258f04ce4563023924e508ee20d878908/stack.yaml#L92

Reply:
The branch was already in the comment, but I've reformatted to look like your example.

Reply:
Ah, silly of me!